source/MRMesh/MRExpected.h

```diff
@@ -32,13 +41,13 @@ inline auto unexpected( E &&e )
 namespace MR
 {
 
-#if ( __cpp_lib_expected >= 202211  ||  defined( MR_DOT_NET_BUILD ) )
+#if MR_USE_STD_EXPECTED || defined(MR_DOT_NET_BUILD)
 
 template<class T, class E = std::string>
 using Expected = std::expected<T, E>;
 
 template <class E>
-inline auto unexpected( E &&e )
+MR_BIND_IGNORE inline auto unexpected( E &&e )
```

Comment:
can we ignore it automatically (like Json::Value functions) ?

Reply:
I don't see any obvious criteria (that we could use to guess that this function is unnecessary).